package.json

```diff
@@ -51,7 +51,7 @@
     "start-server-dev": "cross-env NODE_ENV=development ./bin/cncjs --port 8000 -m /widget:https://cncjs.github.io/cncjs-widget-boilerplate/v1/",
     "lint": "concurrently --kill-others-on-fail --names \"i18nlint,eslint,stylint\" \"yarn run i18nlint\" \"yarn run eslint\" \"yarn run stylint\"",
     "eslint": "eslint --color --ext .js --ext .jsx *.js src scripts test",
-    "eslint-debug": "DEBUG=\"eslint:glob-util,eslint:config-*,eslint:cli*\" eslint --cache --color --quiet --ext .js --ext .jsx *.js src scripts test",
+    "eslint-debug": "set DEBUG=\"eslint:glob-util,eslint:config-*,eslint:cli*\" eslint --cache --color --quiet --ext .js --ext .jsx *.js src scripts test",
```

Comment:
I had to do this to enable me to push to my branch. Guessing it might be due to running windows? 🤷